Core/Blob/Extensions/BlobServiceBuilderExtensions.cs

```diff
@@ -38,12 +39,37 @@ public static class BlobServiceBuilderExtensions
 
             builder
                 .AddProgramInitializer<BlobInitializerService>()
-                .AddAssemblies(typeof(AutoMapperProfile).Assembly)
+                .AddAssemblies(typeof(Data.AutoMapperProfile).Assembly)
                 .AddDatabase<BlobDbContext>(connectionStringName, connectionStringPassword)
+                .AddAuditTrailing()
```

Comment:
I wouldn't add  the audit trailing here. Maybe, on another project the audit trailing won't be used.
It's enough to add it in your app startup and it works
![image](https://user-images.githubusercontent.com/40164286/180191674-ee6e8b9f-ae47-4084-9a12-7e9b3e049e0c.png)

Reply:
Sharp, fixed this.